### DISCUSSION TITLE
QField Cloud Team Work

### DISCUSSION THREAD

**Discussion by Deucalion12** [opengisch/QField#3936] [Q&A]
Hi, as far as I have understood this is how QField Cloud Collaboration Works:

1) I upload project from QGIS using QFIELDSYNC plugin.
2) I download the project locally on my phone (through a QField Cloud account).
3) After I make changes, I either "Push Changes" (to only upload the changes I made) or "Synchronize" to download and upload all the changes. 

So the question is, is there a way to collaborate without downloading the project locally? 
I do not want to have to upload and download projects mid work, while I want all users to be able to see the changes live, so that they do not overwrite each other. As far as I have understood, this can't be done regarding the Desktop QGIS, but can mobile phone field users see each other's updates live?

**Comment by ZsanettMed** [comment]
Hi,

yes it is possible with the live Postgis connection, you can find more informtion here: https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#example-workflow-postgis
